test/extensions/filters/network/redis_proxy/mocks.h

```diff
@@ -4,18 +4,25 @@
 #include <list>
 #include <string>
 
-#include "envoy/redis/command_splitter.h"
-#include "envoy/redis/conn_pool.h"
-
-#include "common/redis/codec_impl.h"
+#include "extensions/filters/network/redis_proxy/codec_impl.h"
+#include "extensions/filters/network/redis_proxy/command_splitter.h"
+#include "extensions/filters/network/redis_proxy/conn_pool.h"
 
 #include "test/test_common/printers.h"
 
 #include "gmock/gmock.h"
 
 namespace Envoy {
-namespace Redis {
+namespace Extensions {
+namespace NetworkFilters {
+namespace RedisProxy {
+
+/**
+ * Pretty print const RespValue& value
+ */
 
+void PrintTo(const RespValue& value, std::ostream* os);
+void PrintTo(const RespValuePtr& value, std::ostream* os);
```

Comment:
Do these belong in mocks, or just stuck here for convenience?  I guess it's not much worse than having = here...

Reply:
Previously, this code was just being included into the mocks, so it's no different than it was before. I agree this is not optimal. In general, I would like to kill all the printers stuff as we have already discussed and just put printing with product code and have it be tested when implemented (https://github.com/envoyproxy/envoy/issues/2205).